### PR TITLE
[RFR] Fix SVG viewport

### LINKED ISSUE #96: Some data are outside the SVG viewport
For instance, currently:

image

If we put another CSS rule on the SVG element (`overflow: visible;`), we have the correct view:

image

SVG has a red background on the second screenshot.

We need to draw data visible in the viewport, and only them.

### PR DESCRIPTION
**Pull request by jpetitcolas**
Fixes #96.

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Switching to RFR.

**Comment by jpetitcolas** [comment]
And tests are green!

File: lib/drawer/index.js [hunk 1]
```diff
@@ -13,7 +13,12 @@ export default (svg, dimensions, scales, configuration) => {
         .attr('id', 'drops-container-clipper')
         .append('rect')
         .attr('id', 'drops-container-rect')
-        .attr('width', dimensions.width)
+        .attr('transform', `translate(0, 0)`)
```
Comment (Kmaschta) [inline lib/drawer/index.js]: Why ?

**Comment by jpetitcolas** [comment]
Review applied. Good catch!

**Pull request merged**